Apps/AdminWebClient/src/ClientApp/src/components/core/CommunicationTable.vue

```diff
@@ -59,7 +62,7 @@ import { Component, Vue, Watch, Emit } from "vue-property-decorator";
 import { SERVICE_IDENTIFIER } from "@/plugins/inversify";
 import container from "@/plugins/inversify.config";
 import BannerFeedback from "@/models/bannerFeedback";
-import Communication from "@/models/communication";
+import Communication from "../../models/communication";
```

Comment:
why not @?

Reply:
not sure if this exists across all IDEs but when I have "@" instead of ../../ it picks up the communication model in WebClient and throws a bunch of errors

Reply:
on second thought, I'm going to rename to remove confusion